jobs/JGFS_ATMOS_AWIPS_20KM_1P0DEG

```diff
@@ -47,21 +47,22 @@ export pgmout=OUTPUT.$$
 
 ########################################################
 # Execute the script.
-${HOMEgfs}/scripts/exgfs_atmos_awips_20km_1p0deg.sh ${fcsthrs}
+"${HOMEgfs}/scripts/exgfs_atmos_awips_20km_1p0deg.sh" "${fcsthrs}"
 export err=$?; err_chk
 ########################################################
 
 ############################################
 # print exec I/O output
 ############################################
-if [ -e "${pgmout}" ] ; then
-  cat ${pgmout}
+if [[ -e "${pgmout}" ]] ; then
+  cat "${pgmout}"
 fi
 
 ###################################
 # Remove temp directories
 ###################################
-if [ "${KEEPDATA}" != "YES" ] ; then
-  rm -rf ${DATA}
+cd "${DATAROOT}" || pass
```

Comment:
I think this one, while it should never fail, should be non-fatal. The only thing left is cleaning up `${DATA}`, which doesn't really require we move out of the `${DATA}` directory to do it. I only added it because I was getting some messages about being in a non-existent directory until the next script did a `cd`.